IceWhaleTech/CasaOS-AppManagement
# feat: add WEBUI_PORT interpolation

# What does the PR do?
The PR adds a new interpolation value. the interpolation is different from base interpolation. It only interpolates once when installing a new app and it will be written in yaml file(/var/lib/casasos/app).

# Test Steps
1. register a new app store https://github.com/CorrectRoadH/CasaOS-AppStore
2. install `usememos` of the store.(the name is difference from official store)
3. open the app and see the URL. we can see the port is assigned by casasos.
